docs/partitions.md

```diff
@@ -1,226 +1,59 @@
-# Partitions
+# Partition Healing
 
-In the original implementation of ringpop, if a cluster is split to multiple
-partitions, nodes in each partition declare each other as faulty, and afterward
-will no longer communicate. Ringpop implemented support for merging the
-partitions, which we call `healing`.
+In the original implementation of ringpop, if a cluster is split to multiple partitions, nodes in each partition declare each other as faulty, and afterward will no longer communicate. Ringpop implemented support for merging the partitions, which we call `healing`.
```

Comment:
Maybe also add something like:
If the network partition is healed, the ringpop cluster is still partitioned into smaller ringpop clusters that don't ping each other. We introduce Partition Healing to merge these smaller clusters back into one healthy ringpop cluster.